tools/cloud-build/daily-tests/ansible_playbooks/base-integration-test.yml

```diff
@@ -143,7 +143,6 @@
         ghpc_stderr: "{{ deployment.stderr | replace('\n',' ') }}"
 
     - name: Gather logs
-      failed_when: false
```

Comment:
The idea here is what? That the imported task cannot fail so we no longer need to protect it here? Technically the assert at the beginning could still trigger a failure. Maybe we just leave this in. Thoughts?

Reply:
I see, I had thought this was on develop, but now I see it was added in this pr.